xla/tests/onednn_layer_norm_test.cc

```diff
@@ -92,66 +92,134 @@ TEST_F(LayerNormTest, SimpleTest) {
   )");
 }
 
-TEST_F(LayerNormTest, SimpleTestBF16) {
+TEST_F(LayerNormTest, LayerNormTest0_BF16) {
   const char* layer_norm_module_str = R"(
-  HloModule layer_norm_bf16.test, entry_computation_layout={(f32[768]{0}, f32[768]{0}, bf16[16,128,768]{2,1,0})->bf16[16,128,768]{2,1,0}}, allow_spmd_sharding_propagation_to_output={true}
+  HloModule layer_norm.test, entry_computation_layout={(bf16[84,197,768]{2,1,0}, f32[768]{0}, f32[768]{0})->bf16[84,197,768]{2,1,0}}
```

Comment:
Since the module strings for BF16 and FP16 are just the FP32 one with minor modifications, can we have just one base string as a template for all three tests (FP32, BF16, and FP16) and just do some string manipulations to change the dtype / add conversion?

Reply:
I modified the test and tried to address this by creating some common strings for HLO texts that can be used on all dtype tests, let me know if that looks fine.